EIPS/eip-5190.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 5190
+title: Account-bound Finance
+description: A smart contract design pattern in decentralized finance to remove insolvency and arbitrary loss of funds using blockchain technology only. Extension of EIP-4973
+author: Hyungsuk Kang (@hskang9)
+discussions-to: https://ethereum-magicians.org/t/
```

Comment:
This needs to be a link to a discussions thread.

Reply:
https://ethereum-magicians.org/t/pr-5252-discussion-account-bound-finance/10027

Reply:
```suggestion
discussions-to: https://ethereum-magicians.org/t/pr-5252-discussion-account-bound-finance/10027
```